src/main/resources/i18n/messages.properties

```diff
@@ -711,4 +711,5 @@ linksModerationReport=Moderation Report Form
 linksDiscord=Discord
 linksFacebook=Facebook
 linksDevNews=Dev Blog
-linksYoutube=YouTube
\ No newline at end of file
+linksYoutube=YouTube
+gameQuality.nan=-
```

Comment:
not-a-number is a technical term. But from the user experience side it means "undefined" here. I would not call it `nan` in the translation string (think about the translators not knowing what this is)